### PR TITLE
[Added] get dealts data 

### PR DESCRIPTION
**Pull request by dan12411**
新增 get dealts method & data model.

### CODE REVIEW

**Review by nick45chen — APPROVED** [review]

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt [hunk 1]
```diff
@@ -19,4 +20,10 @@ interface IntradayService {
         @Query("symbolId") symbolId: String,
         @Query("apiToken") apiToken: String
     ): Response<Intraday<ChartData>>
+
+    @GET("intraday/dealts")
+    suspend fun getDealts(
+        @Query("symbolId") symbolId: String,
+        @Query("apiToken") apiToken: String
```
Comment (nick45chen) [inline fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt:27]: @dan12411 缺少 parameters

image

Comment (nick45chen) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt:27]: btw  `limit` / `offset` / `oddLot` 為 optional parameter (有預設值)

**Review by nick45chen — CHANGES_REQUESTED** [review]

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt [hunk 2]
```diff
@@ -31,4 +32,16 @@ class FugleHttpRepository {
             }
         }
     }
+
+    suspend fun getDealts(symbolId: String, apiToken: String): Result<DealtsData> {
```
Comment (nick45chen) [inline fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt:36]: 同 https://github.com/Make-a-good-soup/fugle-realtime-kotlin/pull/4#discussion_r1091285375

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt (hunk 1 above)
Comment (nick45chen) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt:27]: nickpick:  補上文件註解說明 https://kotlinlang.org/docs/coding-conventions.html#file-annotations

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt (hunk 2 above)
Comment (nick45chen) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt:36]: nickpick:  補上文件註解說明 https://kotlinlang.org/docs/coding-conventions.html#file-annotations

**Review by nick45chen — APPROVED** [review]

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt (hunk 1 above)
Comment (dan12411) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/remote/services/IntradayService.kt]: 感謝 @nick45chen 已修正 [ec9417f](https://github.com/Make-a-good-soup/fugle-realtime-kotlin/pull/4/commits/ec9417f90da9082d9a0e805dfafdb4867a7bbc65)

File: fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt (hunk 2 above)
Comment (dan12411) [reply fugle_realtime_lib/src/main/java/net/makeagoodsoup/fugle_realtime_lib/core/repository/FugleHttpRepository.kt]: 同上

**Pull request merged**